packages/openneuro-components/src/facets/FacetRadio.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { AccordionTab } from '../accordion/AccordionTab'
+import { AccordionWrap } from '../accordion/AccordionWrap'
+import { RadioGroup } from '../radio/RadioGroup'
+import './facet.scss'
+
+export interface FacetRadioProps {
+  radioArr: {
+    label: string
+    onChange?: React.MouseEventHandler<HTMLInputElement>
+    checked: boolean
+    value: string
+  }[]
+  layout: string
+  name: string
+  accordionStyle: string
+  startOpen: boolean
+  label: string
+  dropdown: boolean
```

Comment:
```suggestion
  dropdown?: boolean
```
Both here and in `AccordionTab`.